src/main/resources/run-config.xml

```diff
@@ -0,0 +1,44 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="run" type="MavenRunConfiguration" factoryName="Maven">
+    <MavenSettings>
+      <option name="myGeneralSettings" />
+      <option name="myRunnerSettings">
+        <MavenRunnerSettings>
+          <option name="environmentProperties">
+            <map>
+              <entry key="INTEGRATION_ID" value="src/main/resources/integration-id.txt" />
```

Comment:
These should not be inside the resources directory because the compiler will now include the keys in the jar when you build the project